WalletWasabi/Tor/TorMonitor.cs

```diff
@@ -169,8 +171,11 @@ protected override async Task ActionAsync(CancellationToken token)
 					try
 					{
 						Logger.LogInfo("Tor cannot access remote host. Test fallback URI.");
-						using HttpRequestMessage request = new(HttpMethod.Get, FallbackBackendUri);
-						using HttpResponseMessage _ = await HttpClient.SendAsync(request, token).ConfigureAwait(false);
+						using HttpRequestMessage request = new(HttpMethod.Get, TestApiUri);
+						using HttpResponseMessage response = await HttpClient.SendAsync(request, token).ConfigureAwait(false);
+
+						// Throws if the HTTP status code is not in range 200-299.
+						response.EnsureSuccessStatusCode();
```

Comment:
Any HTTP response code means Tor is working, it's the server has a problem. Someone must had to create that HTTP response code and if the Wasabi server is down then it may be that Tor is giving us